### PR TITLE
Tests for dom/truncate and linting of the entire tests suite.

### PR DESCRIPTION
**Pull request by mikesherov**

### CODE REVIEW

**Comment by mikesherov** [comment]
@Attamusc how do even?

**Comment by mikesherov** [comment]
You added this label in the time it took me to type `git pr` on my terminal, and then copy/paste the link to the PR and open it in my browser.

**Comment by Attamusc** [comment]
Hahaha I'll never tell ;) but seriously, that's what web hooks are for :D

File: .jshintrc [hunk 1]
```diff
@@ -2,11 +2,6 @@
   "strict": true,
   "unused": true,
 
-  //move to JSCS eventually
```
Comment (Attamusc) [inline .jshintrc]: Oh boy, does that mean this was really moved :D

Comment (mikesherov) [inline .jshintrc]: ya, forever ago.

File: test/specs/util/xhr.js [hunk 2]
```diff
@@ -40,7 +41,7 @@ define(['util/xhr', 'nbd/Promise'], function(xhr, Promise) {
 
       response.then(success, error).then(function() {
         expect(success).toHaveBeenCalled();
-        expect(succes).toHaveBeenCalledWith(text);
```
Comment (mikesherov) [inline test/specs/util/xhr.js]: HOW EVEN EVER DID THIS WORK?

File: test/specs/util/xhr.js [hunk 3]
```diff
@@ -40,7 +41,7 @@ define(['util/xhr', 'nbd/Promise'], function(xhr, Promise) {
...
+        expect(success).toHaveBeenCalledWith(text);
         expect(error).not.toHaveBeenCalled();
       }).finally(done);
```
Comment (mikesherov) [inline test/specs/util/xhr.js]: OH. finally done! Swallow that error friend.

Comment (Attamusc) [inline test/specs/util/xhr.js]: so funny

**Comment by Attamusc** [comment]
:+1:

**Pull request merged**